src/components/Admin/index.jsx

```diff
@@ -269,6 +281,8 @@ class Admin extends React.Component {
     } = this.props;
 
     const { params: { actionSlug } } = match;
+    const { location } = this.props;
+    const filtersActive = location.search;
```

Comment:
should this check be more specfic? meaning if there is anything in the url does that classify right now as filtersActive?

Reply:
Soooo... I debated this one. Right now, we don't need it to be more specific; there's nothing else on this page that uses queries. In the future, we might benefit from it being more specific, or we might want to move this functionality out of the querystring altogether. (I'm in favor of the latter, tbh.)

Reply:
totally! That is in fact what I thought too, could we not achieve with by updating state instead? but perhaps the url is also a requirement if you want to send links etc .. so still keep that maybe? not sure at all, we can talk but does not block this pr I think

Reply:
yeah resolving this one, since we discussed in testing too. right now urls seems like sufficient to do this, perhaps a rework using state + url in the future but not too worried about it right now